### PR TITLE
Add sse kms support

### PR DESCRIPTION
**Pull request by munk**
This addresses https://github.com/spark-redshift-community/spark-redshift/issues/82

### CODE REVIEW

File: src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala [hunk 1]
```diff
@@ -292,5 +293,10 @@ private[redshift] object Parameters {
      * include in the COPY command (e.g. `COPY "PUBLIC"."tablename" ("column1" [,"column2", ...])`)
      */
     def includeColumnList: Boolean = parameters("include_column_list").toBoolean
+
+    /**
+     * The AWS SSE-KMS key to use for encryption during UNLOAD operations instead of AWS's default encryption
```
Comment (jsleight) [inline src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala:298]: travis build failed with `/home/travis/build/spark-redshift-community/spark-redshift/src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala:298: File line length exceeds 100 characters`

**Review by jsleight — COMMENTED** [review]
PR lgmt once travis succeeds.  Thanks for contributing!

File: src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala (hunk 1 above)
Comment (munk) [reply src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala]: This is fixed now.

**Review by jsleight — APPROVED** [review]

**Comment by 88manpreet** [comment]
Thanks for adding the test too.

**Pull request merged**